tests/config-tests.ts

```diff
@@ -15,18 +15,21 @@
  * limitations under the License.
  */
 
-import { setMaxListeners } from '../build/app-common';
+import { set as setEsVersion } from '@balena/es-version';
+// Set the desired es version for downstream modules that support it
+setEsVersion('es2018');
 
-setMaxListeners(35); // it appears that 'nock' adds a bunch of listeners - bug?
+import * as tmp from 'tmp';
+tmp.setGracefulCleanup();
+// Use a temporary dir for tests data
+process.env.BALENARC_DATA_DIRECTORY = tmp.dirSync().name;
+
+import { EventEmitter } from 'events';
+EventEmitter.defaultMaxListeners = 35; // it appears that 'nock' adds a bunch of listeners - bug?
```

Comment:
I changed this to editing `events` directly so we don't load a bunch of the cli in here since in general this file is to set up the test environment prior to actually running any tests/cli code